tests/nodeCrash/nodeCrash_test.go

```diff
@@ -0,0 +1,133 @@
+package tests
+
+import (
```

Comment:
this should be part of longevity, please add the test in testTriggers

Reply:
Thanks for the comments. I have added the test in testTriggers

Reply:
u can delete this